scripts/settings_list.py

```diff
@@ -0,0 +1,53 @@
+import yaml
+import pprint
+
+class SettingsList():
+  list_of_dicts= list();
+
+  def sl_print(self):
```

Comment:
don't need the `sl_` prefix here as these functions are uniquely identified by the class they are members of